docs/src/changelog.md

```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 <!-- The line "## <square-bracket>Unreleased</square-bracket>" is replaced by the release/release.py script with the new release version and release date. -->
 
+## [Unreleased]
+
+### Deployment notes
+
+If your currently installed *Quality-time* version is v4.10.0 or older, please read the v5.0.0 deployment notes first.
+
+### Added
+
+- When changing the status of measurement entities (violations, warnings, issues, etc.) also set the status end date, after which the measurement entity should be reviewed. Each status has an associated default time after which it should be reviewed. The defaults can be changed by expanding the report title and navigating to the 'Desired reaction times' tab. For individual measurement entities, the default can be overridden by first changing the status and then changing or removing the status end date. Closes [#5099](https://github.com/ICTU/quality-time/issues/5099).
```

Comment:
Seems rather long for the changelog, is there no regular documentation about this feature?

Reply:
Added text to docs and made changelog entry briefer.